lib/modules/modules.go

```diff
@@ -66,6 +66,8 @@ type Features struct {
 	Plugins bool
 	// AutomaticUpgrades enables automatic upgrades of agents/services.
 	AutomaticUpgrades bool
+	// IsUsageBased enables some usage-based billing features
+	IsUsageBased bool
```

Comment:
to keep the flag name consistent and more descriptive?
```suggestion
	// IsUsageBasedBilling enables some usage-based billing features
	IsUsageBasedBilling bool
```